### DISCUSSION TITLE
Commit message visibility for auto-merge PRs

### DISCUSSION THREAD

**Discussion by covracer** [community/community#8894] [Pull Requests]
My team uses `.github/pull_request_template.md` and "Squash and merge". As a workaround to the default "Squash and merge" commit messages being useless, we usually copy from the Pull Request description / top comment, that follows the template, and paste into the commit message box.

A colleague recently enabled auto-merge on their pull request, and probably followed that usual practice. But I'd like to double check, and I don't see any way to view their proposed commit message.

**Comment by webjunkie** [comment]
I also have this problem. There is no way to see or check the commit message that will be used, even for myself.

**Comment by Marcin-Lach** [comment]
+1

**Comment by sallner** [comment]
I would also be interested in this feature.

**Comment by Nancy-Robertson** [comment]
We would also love to have this feature in github enterprise

**Comment by alunamun** [comment]
I'd like to have this working too. How do these requests for features work? Does it need many comments? I can get my coworkers "voting" this if that's what it takes.
